source/OptimaJet.DWKit.StarterApplication/Services/BackgroundEmailQueueService.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentEmail.Core;
+using FluentEmail.Core.Interfaces;
+using Microsoft.CSharp.RuntimeBinder;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Options;
+using Optimajet.DWKit.StarterApplication.Models;
+using OptimaJet.DWKit.StarterApplication.Repositories;
+using Serilog;
+
+namespace OptimaJet.DWKit.StarterApplication.Services
+{
+    public class BackgroundEmailQueueService : BackgroundService
```

Comment:
I bet we could replace this background service with:

https://github.com/HangfireIO/Hangfire

```cs
var id = BackgroundJob.Enqueue(() => SendEmailOrSomething(...params));
```

Reply:
I will look into this.
https://developertown.visualstudio.com/SIL%20International%20Scriptoria/_workitems/edit/28239